src/hooks/__tests__/useRemainingRelays.test.ts

```diff
@@ -60,6 +60,25 @@ describe('fetch remaining relays hooks', () => {
       renderHook(() => useRemainingRelaysBySafe())
       expect(mockFetch).toHaveBeenCalledTimes(0)
     })
+
+    it('refetch if the txHistoryTag changes', () => {
+      global.fetch = jest.fn()
+      const mockFetch = jest.spyOn(global, 'fetch')
+
+      const render = renderHook(() => useRemainingRelaysBySafe())
+      expect(mockFetch).toHaveBeenCalledTimes(1)
+
+      jest.spyOn(useSafeInfo, 'default').mockReturnValue({
+        safe: {
+          txHistoryTag: 'new',
+        },
+        safeAddress: SAFE_ADDRESS,
+      } as ReturnType<typeof useSafeInfo.default>)
```

Comment:
👀

Reply:
It requires the type to appease TypeScript.